docker/dockerfile-base

```diff
@@ -0,0 +1,48 @@
+FROM alpine:latest
+
+ENV ELM_VERSION 0.19.1
+ENV SCALA_VERSION 2.12.6
```

Comment:
This seems unused. It shouldn't be necessary to specify anyways. Can it be removed?